addons/interaction/ACE_ZeusActions.hpp

```diff
@@ -33,14 +34,32 @@ class ACE_ZeusActions {
         class remoteControl {
             displayName = "$STR_A3_CfgVehicles_ModuleRemoteControl_F";
             icon = "\A3\Modules_F_Curator\Data\portraitRemoteControl_ca.paa";
-            statement = "_unit = objNull; { if ((side _x in [east,west,resistance,civilian]) && !(isPlayer _x)) exitWith { _unit = _x; }; } forEach (curatorSelected select 0); bis_fnc_curatorObjectPlaced_mouseOver = ['OBJECT',_unit]; (group _target) createUnit ['ModuleRemoteControl_F',[0,0,0],[],0,'NONE'];";
+            condition = QUOTE(-1 < (curatorSelected select 0) findIf {_x isKindOf 'CAManBase'});
+            statement = QUOTE( \
+                private _unit = objNull; \
+                { \
+                    if ( \
```

Comment:
this looks like it should be `findIf`

Reply:
Can't imagine how. Could you explain please?

Reply:
```
{
    if (
        side _x in [east,west,resistance,civilian]
        && !(isPlayer _x)
    ) exitWith { _unit = _x; };
} forEach (curatorSelected select 0); 
```
All this does is find the first unit where the condition returns true. Which is also what findIf does, returns the index of the first element where condition returns true..
```
private _unit = (curatorSelected select 0) param [
    (curatorSelected select 0) findIf { side _x in [east,west,resistance,civilian] && !(isPlayer _x) }
    , objNull];
```

Reply:
I didn't know `param` can handle `-1` as index. Without it this code wouldn't be so elegant.

Reply:
param/params are literally the best commands ever.